{"uuid":"34c6dec9-6d27-4cd7-b9a5-f9f0aacb037f","historyId":"1257fcc384a74315a6f5dfe36418bc81","fullName":"ge.tbc.testautomation.CoreFunctionalityTests.searchTest","labels":[{"name":"package","value":"ge.tbc.testautomation.CoreFunctionalityTests"},{"name":"testClass","value":"ge.tbc.testautomation.CoreFunctionalityTests"},{"name":"testMethod","value":"searchTest"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"second-final-project"},{"name":"subSuite","value":"ge.tbc.testautomation.CoreFunctionalityTests"},{"name":"host","value":"DESKTOP-3QHAGIK"},{"name":"thread","value":"23208@DESKTOP-3QHAGIK.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"searchTest","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"org.apache.ibatis.exceptions.PersistenceException: \r\n### Error querying database.  Cause: com.microsoft.sqlserver.jdbc.SQLServerException: Cannot open database \"TestAutomation\" requested by the login. The login failed. ClientConnectionId:3c98734e-2441-4af4-94d8-51f5ed0cd7ba\r\n### The error may exist in ge/tbc/testautomation/mappers/BookingCaseMapper.java (best guess)\r\n### The error may involve ge.tbc.testautomation.mappers.BookingCaseMapper.findAll\r\n### The error occurred while executing a query\r\n### Cause: com.microsoft.sqlserver.jdbc.SQLServerException: Cannot open database \"TestAutomation\" requested by the login. The login failed. ClientConnectionId:3c98734e-2441-4af4-94d8-51f5ed0cd7ba","trace":"java.lang.RuntimeException: org.apache.ibatis.exceptions.PersistenceException: \r\n### Error querying database.  Cause: com.microsoft.sqlserver.jdbc.SQLServerException: Cannot open database \"TestAutomation\" requested by the login. The login failed. ClientConnectionId:3c98734e-2441-4af4-94d8-51f5ed0cd7ba\r\n### The error may exist in ge/tbc/testautomation/mappers/BookingCaseMapper.java (best guess)\r\n### The error may involve ge.tbc.testautomation.mappers.BookingCaseMapper.findAll\r\n### The error occurred while executing a query\r\n### Cause: com.microsoft.sqlserver.jdbc.SQLServerException: Cannot open database \"TestAutomation\" requested by the login. The login failed. ClientConnectionId:3c98734e-2441-4af4-94d8-51f5ed0cd7ba\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:58)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:159)\r\n\tat org.testng.internal.Parameters.handleParameters(Parameters.java:809)\r\n\tat org.testng.internal.Parameters.handleParameters(Parameters.java:734)\r\n\tat org.testng.internal.invokers.ParameterHandler.handleParameters(ParameterHandler.java:72)\r\n\tat org.testng.internal.invokers.ParameterHandler.createParameters(ParameterHandler.java:51)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:946)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1597)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:739)\r\n\tat org.testng.TestRunner.run(TestRunner.java:614)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\r\n\tat org.testng.TestNG.run(TestNG.java:1079)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\r\nCaused by: org.apache.ibatis.exceptions.PersistenceException: \r\n### Error querying database.  Cause: com.microsoft.sqlserver.jdbc.SQLServerException: Cannot open database \"TestAutomation\" requested by the login. The login failed. ClientConnectionId:3c98734e-2441-4af4-94d8-51f5ed0cd7ba\r\n### The error may exist in ge/tbc/testautomation/mappers/BookingCaseMapper.java (best guess)\r\n### The error may involve ge.tbc.testautomation.mappers.BookingCaseMapper.findAll\r\n### The error occurred while executing a query\r\n### Cause: com.microsoft.sqlserver.jdbc.SQLServerException: Cannot open database \"TestAutomation\" requested by the login. The login failed. ClientConnectionId:3c98734e-2441-4af4-94d8-51f5ed0cd7ba\r\n\tat org.apache.ibatis.exceptions.ExceptionFactory.wrapException(ExceptionFactory.java:30)\r\n\tat org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:156)\r\n\tat org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:147)\r\n\tat org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:142)\r\n\tat org.apache.ibatis.binding.MapperMethod.executeForMany(MapperMethod.java:147)\r\n\tat org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:80)\r\n\tat org.apache.ibatis.binding.MapperProxy$PlainMethodInvoker.invoke(MapperProxy.java:141)\r\n\tat org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:86)\r\n\tat jdk.proxy2/jdk.proxy2.$Proxy20.findAll(Unknown Source)\r\n\tat ge.tbc.testautomation.data.DataSupplier.getBookingTestData(DataSupplier.java:22)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:54)\r\n\t... 24 more\r\nCaused by: com.microsoft.sqlserver.jdbc.SQLServerException: Cannot open database \"TestAutomation\" requested by the login. The login failed. ClientConnectionId:3c98734e-2441-4af4-94d8-51f5ed0cd7ba\r\n\tat com.microsoft.sqlserver.jdbc.SQLServerException.makeFromDatabaseError(SQLServerException.java:270)\r\n\tat com.microsoft.sqlserver.jdbc.TDSTokenHandler.onEOF(tdsparser.java:329)\r\n\tat com.microsoft.sqlserver.jdbc.TDSParser.parse(tdsparser.java:137)\r\n\tat com.microsoft.sqlserver.jdbc.TDSParser.parse(tdsparser.java:42)\r\n\tat com.microsoft.sqlserver.jdbc.SQLServerConnection.sendLogon(SQLServerConnection.java:6888)\r\n\tat com.microsoft.sqlserver.jdbc.SQLServerConnection.logon(SQLServerConnection.java:5434)\r\n\tat com.microsoft.sqlserver.jdbc.SQLServerConnection$LogonCommand.doExecute(SQLServerConnection.java:5366)\r\n\tat com.microsoft.sqlserver.jdbc.TDSCommand.execute(IOBuffer.java:7745)\r\n\tat com.microsoft.sqlserver.jdbc.SQLServerConnection.executeCommand(SQLServerConnection.java:4391)\r\n\tat com.microsoft.sqlserver.jdbc.SQLServerConnection.connectHelper(SQLServerConnection.java:3828)\r\n\tat com.microsoft.sqlserver.jdbc.SQLServerConnection.login(SQLServerConnection.java:3385)\r\n\tat com.microsoft.sqlserver.jdbc.SQLServerConnection.connectInternal(SQLServerConnection.java:3194)\r\n\tat com.microsoft.sqlserver.jdbc.SQLServerConnection.connect(SQLServerConnection.java:1971)\r\n\tat com.microsoft.sqlserver.jdbc.SQLServerDriver.connect(SQLServerDriver.java:1263)\r\n\tat java.sql/java.sql.DriverManager.getConnection(DriverManager.java:682)\r\n\tat java.sql/java.sql.DriverManager.getConnection(DriverManager.java:191)\r\n\tat org.apache.ibatis.datasource.unpooled.UnpooledDataSource.doGetConnection(UnpooledDataSource.java:229)\r\n\tat org.apache.ibatis.datasource.unpooled.UnpooledDataSource.doGetConnection(UnpooledDataSource.java:224)\r\n\tat org.apache.ibatis.datasource.unpooled.UnpooledDataSource.getConnection(UnpooledDataSource.java:97)\r\n\tat org.apache.ibatis.datasource.pooled.PooledDataSource.popConnection(PooledDataSource.java:452)\r\n\tat org.apache.ibatis.datasource.pooled.PooledDataSource.getConnection(PooledDataSource.java:100)\r\n\tat org.apache.ibatis.transaction.jdbc.JdbcTransaction.openConnection(JdbcTransaction.java:145)\r\n\tat org.apache.ibatis.transaction.jdbc.JdbcTransaction.getConnection(JdbcTransaction.java:67)\r\n\tat org.apache.ibatis.executor.BaseExecutor.getConnection(BaseExecutor.java:348)\r\n\tat org.apache.ibatis.executor.SimpleExecutor.prepareStatement(SimpleExecutor.java:89)\r\n\tat org.apache.ibatis.executor.SimpleExecutor.doQuery(SimpleExecutor.java:64)\r\n\tat org.apache.ibatis.executor.BaseExecutor.queryFromDatabase(BaseExecutor.java:336)\r\n\tat org.apache.ibatis.executor.BaseExecutor.query(BaseExecutor.java:158)\r\n\tat org.apache.ibatis.executor.CachingExecutor.query(CachingExecutor.java:110)\r\n\tat org.apache.ibatis.executor.CachingExecutor.query(CachingExecutor.java:90)\r\n\tat org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:154)\r\n\t... 37 more\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1748435328017,"stop":1748435328020}